src/lib/helpers.ts

```diff
@@ -391,3 +391,12 @@ export function generateRandomIntegerInRange(min: number, max: number) {
 export function isSameArray(array1: string[], array2: string[]) {
   return array1.length === array2.length && array1.every((element) => array2.includes(element));
 }
+
+/**
+ * Checks whether the given labware has samples in any of its slots.
+ * @param labware - The labware to check for samples.
+ * @returns `true` if samples are found in any slot, `false` otherwise.
+ */
+export const hasSamples = (labware: LabwareFieldsFragment): boolean => {
+  return labware.slots.some((slot) => slot.samples && slot.samples.length > 0);
+};
```

Comment:
We are currently using helper.ts for generic function. The more appropriate place would be /src/lib/labwareHelper.ts